autometa/common/external/diamond.py

```diff
@@ -462,14 +558,43 @@ def main(args):
         default=200,
         type=int,
     )
-    parser.add_argument("--cpus", help="num cpus to use", default=0, type=int)
-    parser.add_argument("--tmpdir", help="</path/to/tmp/directory>", default=os.curdir)
     parser.add_argument(
-        "--top-pct", help="top percentage of hits to retrieve", default=0.9
+        "--cpus", help="number of processors to use", default=mp.cpu_count(), type=int
+    )
+    parser.add_argument("--tmpdir", help="</path/to/tmp/directory>")
```

Comment:
The help text for this could be more descriptive. It is unclear why specifying a tmp directory is necessary. What is using the tmp directory?

Something like "Path to tmp directory used by diamond (See diamond manual section 3.6 for details)."